packages/apollo/src/controllers/roles-controller.ts

```diff
@@ -80,39 +84,19 @@ export module RoleController {
     .post("/grant")
     .use(authenticated(null))
     .use(authorized(Perm.MANAGE_ROLES))
-    .use(Parser.body(t.type({ memberId: t.string, roles: t.array(t.string) })))
+    .use(Parser.body(t.type({ userId: t.string, roles: t.array(t.string) })))
     .handler(async ({ body }) => {
-      const branchRoles = (
-        await prisma.branchMember.findUnique({
-          where: { id: body.memberId },
-          select: {
-            branch: {
-              select: {
-                roles: {
-                  select: {
-                    id: true,
-                  },
-                },
-              },
-            },
-          },
-        })
-      )?.branch.roles.map((role) => role.id);
-
-      const user = await prisma.branchMember.update({
-        where: { id: body.memberId },
+      const user = prisma.user.update({
+        where: { id: body.userId },
         data: {
           roles: {
-            set: body.roles
-              .filter((role) => branchRoles?.includes(role))
-              .map((r) => ({
-                id: r,
+            createMany: {
```

Comment:
This might cause problems with users who have existing roles which someone is trying to add.  I need to fix this